src/utilities/updateBrowserUrl.ts

```diff
@@ -0,0 +1,39 @@
+import { isBrowser } from '@/utilities/isBrowser'
+
+type UpdateBrowserUrlOptions = {
+  replace?: boolean,
+}
```

Comment:
I have a virtually identical type in my history service. Is it worth trying to combine these?